couchbase/conf.yaml.example

```diff
@@ -4,6 +4,7 @@ instances:
   - server: http://localhost:8091
     user: Administrator
     password: password
+    # query_monitoring_url: http://localhost:8093 # Url to get query monitoring stats, available since couchbase 4.5. See https://developer.couchbase.com/documentation/server/4.5/tools/query-monitoring.html
```

Comment:
Is there any way to determine this programmatically? If not that's fine, but it's definitely better to do so when possible.